integration-tests/micrometer/src/test/java/org/apache/camel/quarkus/component/micrometer/it/MicrometerTest.java

```diff
@@ -207,4 +207,46 @@ public void testJMXQuarkusDomain(String domain) throws Exception {
                 .getValue();
         assertTrue(classes > 1);
     }
+
+    @Test
+    @DisabledOnIntegrationTest("https://github.com/apache/camel-quarkus/issues/5209")
+    public void testJMXHistory() throws Exception {
+        //send a message to init history for the route with id `jmxHistory`
+        RestAssured.get("/micrometer/sendJmxHistory")
+                .then()
+                .statusCode(200);
+
+        String contextManagementName = RestAssured.get("/micrometer/getContextManagementName")
+                .then()
+                .statusCode(200)
+                .extract().body().asString();
+
+        // get the message history service using JMX
+        String name = String.format("org.apache.camel:context=%s,type=services,name=MicrometerMessageHistoryService",
+                contextManagementName);
+
+        MBeanServer mBeanServer = ManagementFactory.getPlatformMBeanServer();
+
+        ObjectName on = ObjectName.getInstance(name);
+
+        //return json result
+        String json = (String) mBeanServer.invoke(on, "dumpStatisticsAsJson", null, null);
+
+        assertTrue(json.contains("jmxHistory"));
+
+    }
+
```

Comment:
Is it correct reading that `testJMXHistory` is checking that the service could be accessed through JMX ?
And that `testHistory` is ensuring that the service could be accessed through the camel context ?

If yes, what about renaming both tests ala:
 + micrometerHistoryShouldBeAvailableThroughJMX
 + micrometerHistoryShouldBeAvailableThroughCamelContext

If make sense.

Reply:
Yeah, if I understand the intent correctly, there's a check to ensure the relevant MBeans are published and metrics are being recorded. And another test to get it directly from the `CamelContext`. I can rename the test methods.

Maybe after #5209 these could be combined into a single test. Checking that the bits are available in JMX is probably enough for the native / non-native case.

Reply:
Test renamed as requested.